### ISSUE TITLE
Forest in Outpost level not appearing in Ubuntu

### ISSUE THREAD

**Issue by eightyeight** [GarageGames/Torque3D#1287]
Level loads, but no forest trees exist. Relevant extract from `console.log`:

```
Executing levels/Outpost.mis.
Failed to create resource: [levels/outpost.forest]
levels/Outpost.mis (0): Register object failed for object theForest of class Forest.
```

Putting this here to remind myself to look into it, and see if anyone else has the issue.

**Comment by LuisAntonRebollo** [comment]
Can be a case-sensitive problem in the file name.

```
Executing levels/Outpost.mis.
Failed to create resource: [levels/outpost.forest]
```

**Comment by eightyeight** [comment]
:sob: I'll check it later on.

**Comment by eightyeight** [comment]
Okay, that was a problem, but a separate one. Edited the post to remove those parts - only the first three lines are relevant to the forest.

**Comment by eightyeight** [comment]
@Azaezel, you were right. When you posted that line with the `defaulttree.dae`, I thought you were referring to the _other_ place that line exists, which is in some random TSStatic that was also causing console errors, which I had fixed, but hadn't fixed in the forest item definition.

As for the source of the crash, the problem originates in [tsForestItemData.h](https://github.com/GarageGames/Torque3D/blob/b30b5d907623a89ddb812992de1b0c588f0f41cf/Engine/source/forest/ts/tsForestItemData.h#L91):

```cpp
   const Box3F& getObjBox() const { return mShape ? mShape->bounds : Box3F::Invalid; }
```

When the shape file it not found, every item gets given an invalid object box. Then, when the forest cell goes to [update its bounds](https://github.com/GarageGames/Torque3D/blob/b30b5d907623a89ddb812992de1b0c588f0f41cf/Engine/source/forest/forestCell.cpp#L180), it tries to update with all these invalid boxes.

I've made two PRs from this. #1307 fixes the default bounds issue. If you're missing a shape file, you really don't want a crash, you want a console error, which you do currently get. #1308 fixes the filename case :p.

Oh, and #1309 adds more detail to that fatal assert we were looking at :p.
